patroni/postgresql/config.py

```diff
@@ -325,7 +325,7 @@ class ConfigHandler(object):
         'track_commit_timestamp': ('off', _bool_validator, 90500),
         'max_replication_slots': (10, IntValidator(min=4), 90400),
         'max_worker_processes': (8, IntValidator(min=2), 90400),
-        'wal_log_hints': ('on', _bool_is_true_validator, 90400)
+        'wal_log_hints': ('on', _bool_validator, 90400)
```

Comment:
https://github.com/zalando/patroni/blob/ff31f45226f03df70e2f4243e84308b5e7138994/patroni/postgresql/config.py#L1104

we should also remove it from here, as it is now configurable

Reply:
Thx for review :)